core/spec/models/spree/shipment_spec.rb

```diff
@@ -750,4 +750,18 @@
       end
     end
   end
+
+  describe ".by_store" do
+    it "returns shipments by store" do
+      olivanders_store = build(:store, name: 'Olivanders')
```

Comment:
This is nitpicking but is there a reason why we need to `build` these? It seems a little confusing to me that we `build` instead of `create`  when we need them all to be persisted for the spec to work.